package.json

```
@@ -54,5 +54,8 @@
   "peerDependencies": {
     "typescript": "^5.0.0"
   },
-  "dependencies": {}
+  "dependencies": {
+    "axios": "^1.7.9",
+    "bun": "^1.2.2"
+  }
```

```suggestion
  "dependencies": {}
```

Deps should be like before since we need to minimise cli size as much possible, i think we don't need axios or bun you can use already present ky or node fetch

---

Sure thanks @ArnavK-09 updated